src/workerd/server/workerd.c++

```diff
@@ -848,9 +850,11 @@ public:
       }
     }
 
-    KJ_IF_SOME(c, config) {
-      Autogate::initAutogate(c);
-    }
+    // We'll fail at getConfig() if there are multiple top level Config objects.
```

Comment:
Does workerd fail somewhere else if this is the case already? or is this a regression that is being introduced by this PR?

Reply:
The error message suggests that there is a code path that handles this, but as far as I can tell that code path does not exist and this is not a regression.